.gitlab-ci.yml

```diff
@@ -120,7 +120,7 @@ variables:
   MACOS_S3_BUCKET: dd-agent-macostesting
   WIN_S3_BUCKET: dd-agent-mstesting
   PROCESS_S3_BUCKET: datad0g-process-agent
-  BUCKET_BRANCH: nightly # path inside the staging s3 buckets to release to: 'nightly', 'oldnightly', 'beta' or 'stable'
+  BUCKET_BRANCH: dev # path inside the staging s3 buckets to release to: 'dev', 'nightly', 'oldnightly', 'beta' or 'stable'
```

Comment:
Does it change the default behaviour? As dev is also the default setting below, can you confirm all related to nightly correctly set this variable to target nightly?

Reply:
I think it has been updated, as mentionned in the `Additional Notes` part